### DISCUSSION TITLE
Integração do app rede-cnpj com bcadastros

### DISCUSSION THREAD

**Discussion by andersonvass** [rictom/rede-cnpj#29] [General]
Bom dia @rictom .

Fizemos recentemente um convênio com a SERPRO para termos acesso aos dados cadastrais do bcadastros (https://loja.serpro.gov.br/b-cadastros). Nele temos as informações de CPF e CNPJ, com atualização frequente.

Assim, tenho as seguintes dúvidas:

1) Há algum script que faça o apontamento para as tabelas do bcadastros?

2) Senão houve um script, poderia dizer, em linhas gerais, o que seria necessário alterar para realizar esse apontamento?

No aguardo

Até mais

**Comment by rictom** [comment]
Olá @andersonvass,

Me encaminhe um email para rictomgit@gmail.com e me explica melhor o seu problema.

**Comment by danielqbarbosa** [comment]
Bom dia.
Tenho o mesmo interesse.
Acredito que muita gente terá essa demanda. O bCadastro é um produto recente do Serpro, que está substituindo antigas ferramentas, como o Hod.

> Bom dia @rictom .
> 
> Fizemos recentemente um convênio com a SERPRO para termos acesso aos dados cadastrais do bcadastros (https://loja.serpro.gov.br/b-cadastros). Nele temos as informações de CPF e CNPJ, com atualização frequente.
> 
> Assim, tenho as seguintes dúvidas:
> 
> 1. Há algum script que faça o apontamento para as tabelas do bcadastros?
> 2. Senão houve um script, poderia dizer, em linhas gerais, o que seria necessário alterar para realizar esse apontamento?
> 
> No aguardo
> 
> Até mais
